com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -156,19 +178,45 @@ void OnEnable()
 
         public void CreateGUI()
         {
-            m_SerializedObject = new SerializedObject(this);
-            converterEditorAsset.CloneTree(rootVisualElement);
+            if (m_ConverterContainers.Any())
+            {
+                m_SerializedObject = new SerializedObject(this);
+                converterEditorAsset.CloneTree(rootVisualElement);
 
+                rootVisualElement.Q<DropdownField>("conversionsDropDown").choices = m_ContainerChoices;
+                rootVisualElement.Q<DropdownField>("conversionsDropDown").index = m_ContainerChoiceIndex;
+                m_CurrentContainer = m_Containers[m_ContainerChoiceIndex];
+                ReCreateUI();
+
+                var button = rootVisualElement.Q<Button>("convertButton");
+                button.RegisterCallback<ClickEvent>(Convert);
+                button.SetEnabled(false);
+
+                var initButton = rootVisualElement.Q<Button>("initializeButton");
+                initButton.RegisterCallback<ClickEvent>(InitializeAllActiveConverters);
+            }
+        }
+
+        void ReCreateUI()
+        {
+            m_SerializedObject.Update();
             // This is temp now to get the information filled in
-            RenderPipelineConverterContainer converterContainer =
-                (RenderPipelineConverterContainer)Activator.CreateInstance(m_ConverterContainers[0]);
-            rootVisualElement.Q<Label>("conversionName").text = converterContainer.name;
-            rootVisualElement.Q<TextElement>("conversionInfo").text = converterContainer.info;
+            rootVisualElement.Q<DropdownField>("conversionsDropDown").RegisterCallback<ChangeEvent<string>>((evt) =>
+            {
+                m_ContainerChoiceIndex = rootVisualElement.Q<DropdownField>("conversionsDropDown").index;
+                m_CurrentContainer = m_Containers[m_ContainerChoiceIndex];
```

Comment:
Do we need a safety check if the choice is within valid range?

Reply:
OK for now